### ISSUE TITLE
After waking from sleep, title bar is hidden but window size is offset

### ISSUE THREAD

**Issue by rcywongaa** [pixel-saver/pixel-saver#211]
After waking from sleep and logging in, the title bar is hidden but the shadow of the title bar is still there (see image).  Also, the window size is offset as though there is still the title bar

![image](https://user-images.githubusercontent.com/7285669/85924302-1003b280-b8c4-11ea-8451-cb3ec350eec5.png)

Also happens with the [no-title-bar extension](https://github.com/franglais125/no-title-bar/issues/123)

**Comment by jobs-git** [comment]
yes same, even when its not sleep, just switch user and it will appear, no-tittle-bar also has this issue.
